ayekan/fluentbit/daemonset-config.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-bit-config
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush        1
+        Log_Level    debug
+
+    [INPUT]
+        Name              tail
+        Path              /var/log/containers/*.log
+        Parser            docker
```

Comment:
Has this been tested as working?

We are using json-file as the log type and it looks like the time format might not match what is being expected.  E.g.:
```
{"log":"[2024/01/25 17:00:11] [ warn] [engine] failed to flush chunk '1-1706202010.617218115.flb', retry in 7 seconds: task_id=1, input=tail.0 \u003e output=opensearch.0 (out_id=0)\n","stream":"stderr","time":"2024-01-25T17:00:11.80843373Z"}
```

Reply:
I might drop this file all together. There is an option to deploy this with a helm chart as well (aside from the static k8s manifests and apparently there is a fluentbit-operator as well ...) and the testing currently uses the helm chart for cluster deployment.